CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Configure `gsoci.azurecr.io` as the default container image registry.
 
+### Fixed
+
+- Remove duplicated team label.
+
```

Comment:
This is in the wrong place, it should be in the `Unreleased` block.

Reply:
Ah, yeah, my bad, thanks for catching this 🤝